gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -394,6 +394,9 @@ bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow
       R__ERROR_HERE("WebDisplay") << "Neither Qt5 nor CEF libraries arenot found";
       return false;
    }
+#else
+   // just avoid compiler warnings
+   static_cast<void>(is_local);
```

Comment:
This look weird to me... `is_local` is declared as `bool`, right?

Reply:
It is just a way to avoid compiler warnings.